controller/accounts.js

```diff
@@ -42,6 +73,38 @@ router.get('/graphs', function(req, res){
   }
 });
 
+// post sprint name to change the data going to the graph
+router.post('/burndown', function(req, res){
+  if(!sess || !sess.username){
+    res.redirect('/');
+  } else {
+    console.log(req.body.sprint);
+    let burndown;
+    let error;
+    //wasn't failing gracefully when typo in sprint name try catch to handel it.
+    try {
+      burndown = kujiraDataMiner.burndownReportData(message, req.body.sprint);
+      error = 'Success found '+req.body.sprint;
+    }catch(err){
+      error = 'No such Sprint named '+req.body.sprint;
+    }
+    fs.writeFile('./public/js/burndown.json',  JSON.stringify(burndown, null, 4), function(err){
```

Comment:
No need the graph data to clear if bad data . Won't do this as part of the try catch.